frontend/packages/component-library/src/dropdown/CustomDropdown.tsx

```diff
@@ -239,8 +251,10 @@ const CustomDropdown: React.FunctionComponent<CustomDropdownProps> = ({
           <FontAwesomeV6Icon iconStyle="solid" iconName="chevron-down" />
         </button>
       )}
-      {/** Dropdown menu content is rendered here as children props*/}
-      {children}
+      <div className={moduleStyles.dropdownMenuContainer}>
```

Comment:
This is added to `CustomDropdown` from callers because it is duplicated in each call of `CustomDropdown` and `CustomDropdown` does not show anything when it is missing.